rust_src/src/process.rs

```diff
@@ -169,7 +171,7 @@ pub fn process_id(process: LispProcessRef) -> Option<EmacsInt> {
 /// deleted or killed.
 #[lisp_fn]
 pub fn get_buffer_process(buffer_or_name: Option<LispBufferOrName>) -> Option<LispProcessRef> {
-    get_buffer_process_internal(buffer_or_name.and_then(|b| b.into()))
+    get_buffer_process_internal(buffer_or_name.and_then(Into::into))
```

Comment:
Looks a bit odd. I couldn't find another way of doing this.

Reply:
`Option<LispProcessRef>::from` would work right?

Perhaps this is a place we have used Convert too much and it should be `LispBufferOrName::to_process()`.